django_project/base/templates/project/new_detail.html

```diff
@@ -194,6 +194,36 @@ <h6><span>No teams are defined, but you can </span>
                     {% endif %}
                 {% endif %}
             </div>
+            <div class="panel panel-default custom-panel clickable-panel" onclick="window.location='section/list/';">
+                <h3>Lessons</h3>
+                {% if sections %}
+                    <h6 class="text-muted">
+                    Learn and learn
+                  </h6>
+                    <div class="content-list">
+                        {% for section in sections %}
+                            <h5>
+                                <span>
+                                    <a href="">
+                                        {{ section.name }}
+                                    </a>
+                                </span>
+                            </h5>
+                        {% endfor %}
+                    </div>
+                    {% else %}
+                    {% if user.is_staff or user == project.owner %}
```

Comment:
We should add a new project role here for lessons_manager...and make it editable by staff, lessons manager or project owner...

Reply:
I will add to the ticket, see: https://github.com/kartoza/projecta/issues/657#issuecomment-356860151